conf/variant/common/bblayers.conf

```diff
@@ -1,3 +1,6 @@
+LAYERSERIES_CORENAMES_append = " rocko"
```

Comment:
Maybe a short comment on why this is done, since I guess it's quite unusual to change another layers config.

Reply:
(The comment above is meant for both lines here, not just the CORENAMES thing)

Reply:
Oops I forgot to add my comment. I had it in another file, then moved it to this file.